### PR TITLE
Custom config for android-add-ons-extension-process

### PR DESCRIPTION
**Pull request by scholtzan**
Adding memory metrics for `android-add-ons-extension-process` rollout

### CODE REVIEW

File: opmon/android-add-ons-extension-process.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[project]
+
+name = "Android Add Ons Extension Process"
+platform = "fenix"
+metrics = [
+  "storage_stats_app_bytes",
+  "storage_stats_cache_bytes",
+  "storage_stats_data_dir_bytes"
```
Comment (scholtzan) [inline opmon/android-add-ons-extension-process.toml:8]: These are some of the memory related metrics available on Fenix and that are defined here: https://github.com/mozilla/metric-hub/blob/972480c8fa975331f3edf625a2e68d5e63581861/definitions/fenix.toml#L192-L214

Feel free to add more metrics. There are docs available here: https://docs.telemetry.mozilla.org/cookbooks/operational_monitoring.html

**Comment by scholtzan** [comment]
cc @mixedpuppy

File: opmon/android-add-ons-extension-process.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+[metrics]
+
+[metrics.performance_page_non_blank_paint]
+data_source = "metrics"
+select_expression = "ARRAY_AGG(metrics.timing_distribution.performance_page_non_blank_paint IGNORE NULLS)"
+friendly_name = "Page Non Blank Paint"
+description = "The time between navigationStart and the first non-blank paint of a foreground root content document, in milliseconds."
+type = "histogram"
```
Comment (scholtzan) [inline opmon/android-add-ons-extension-process.toml:25]: Adding `performance_page_non_blank_paint` here for now. We can make it available across all configs as well if this is something that will be used in other rollouts as well

Comment (mixedpuppy) [reply opmon/android-add-ons-extension-process.toml:25]: It will be used across rollouts we're doing.

File: definitions/fenix.toml [hunk 3]
```diff
@@ -133,6 +133,15 @@ description = "Time in milliseconds from responseStart to loadEventStart for the
...
 
+
+[metrics.performance_page_non_blank_paint]
+data_source = "metrics"
+select_expression = "ARRAY_AGG(metrics.timing_distribution.performance_page_non_blank_paint IGNORE NULLS)"
+friendly_name = "Page Non Blank Paint"
+description = "The time between navigationStart and the first non-blank paint of a foreground root content document, in milliseconds."
+category = "performance"
+type = "histogram"
+
```
Comment (scholtzan) [inline definitions/fenix.toml:144]: alright, any metric definition in this file can be referenced globally

**Pull request merged**